submissions/Rybchynskyi/app-Popup/index.html

```diff
@@ -0,0 +1,161 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="header__row">
+            <div class="header__menu">
+                <a class="link" href="">Mail</a>
+            </div>
+            <div class="header__menu">
+                <a class="link" href="">Images</a>
+            </div>
+            <div class="header__menu for_popup">
+                <label class="check_1">
+                    <img class="header__img__menu" src="img/popup-button.png" alt="menu">
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)